Mathlib/Algebra/Star/BigOperators.lean

```diff
@@ -25,5 +25,5 @@ theorem star_prod [CommMonoid R] [StarMul R] {α : Type*} (s : Finset α) (f : 
 
 @[simp]
 theorem star_sum [AddCommMonoid R] [StarAddMonoid R] {α : Type*} (s : Finset α) (f : α → R) :
-    star (∑ x in s, f x) = ∑ x in s, star (f x) := (starAddEquiv : R ≃+ R).map_sum _ _
+    star (∑ x in s, f x) = ∑ x in s, star (f x) := map_sum (starAddEquiv : R ≃+ R) _ _
```

Comment:
Why didn't this code generate warning? Otherwise LGTM, these lemmas were deprecated for almost a year.

Reply:
Forgot:
bors d+